pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -438,9 +435,9 @@ async def subscribe(subscription: str,  # pylint: disable=too-many-locals
                                          return_when=asyncio.FIRST_COMPLETED)
             for task in done:
                 task.result()
-            raise Exception('A subscriber worker shut down unexpectedly!')
+            raise Exception('a subscriber worker shut down unexpectedly')
         except (asyncio.CancelledError, Exception) as e:
-            log.info('Subscriber exited', exc_info=e)
+            log.info('subscriber exited', exc_info=e)
```

Comment:
While we're here, thoughts on bumping this to `WARNING`? It has been very useful in the past for debugging shutdowns.

Reply:
Yeah, good call. Probably shouldn't be swallowing this stack trace!